src/binaryen-c.h

```diff
@@ -1041,6 +1041,12 @@ BinaryenCallIndirectGetTarget(BinaryenExpressionRef expr);
 BINARYEN_API void
 BinaryenCallIndirectSetTarget(BinaryenExpressionRef expr,
                               BinaryenExpressionRef targetExpr);
+// Gets the table name of a `call_indirect` expression.
+BINARYEN_API const char*
+BinaryenCallIndirectGetTableName(BinaryenExpressionRef expr);
+// Sets the table name of a `call_indirect` expression.
+BINARYEN_API void BinaryenCallIndirectSetTableName(BinaryenExpressionRef expr,
+                                                   const char* table);
```

Comment:
I think it would be reasonable to require a table name in the CallIndirect constructor function as well, even though that would be a breaking change. We would just have to mention it in the change log.

Reply:
This was my initial approach. But then I changed it to avoid a breaking change. I agree that the assumption makes a lot of things much easier!